### DISCUSSION TITLE
{"error":"unauthorized","reason":"You are not a server admin."}

### DISCUSSION THREAD

**Discussion by ArunGust** [apache/couchdb#2995] [Q&A] [answered]
[NOTE]: # ( ^^ Provide a general summary of the issue in the title above. ^^ )

## Description

I have setup admin user while installation and still getting an error when accessing http://admin:password@localhost:5984/_all_dbs

**Comment by janl** [comment]
make sure the password is correct. Without any extra info, we can’t really help here.

**Comment by janl** [comment, marked as answer]
Ah, dupe of https://github.com/apache/couchdb/discussions/2994
